### ISSUE TITLE
Parcellite not showing up

### ISSUE THREAD

**Issue by sverhagen** [rickyrockrat/parcellite#30]
I _think_ Parcellite may be something I want, but I'm unable to run it. I'm on Ubuntu 18.04.2 LTS and installed Parcellite using APT. There's an application I can start, and when I do, it shows up in my processes:

    $ ps aux | grep parcellite
    sander   20201  0.3  0.0 353100 19980 tty2     Sl+  01:59   0:00 parcellite

However, I believe I should be seeing an icon for it at the top of my screen, and it's not there.

This is in `/var/log/syslog`:

    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Looking in '/etc/xdg/xdg-ubuntu/parcellite/parcelliterc'
    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Looking in '/etc/xdg/parcellite/parcelliterc'
    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Looking in '/etc/xdg/xdg-ubuntu/parcellite/parcelliterc'
    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Looking in '/etc/xdg/parcellite/parcelliterc'
    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Flag 0x0001, status 0, EXIT 1 STAT 0
    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Looking in '/etc/xdg/xdg-ubuntu/parcellite/parcelliterc'
    Feb 18 01:59:33 sander-laptop3 parcellite.desktop[20201]: Looking in '/etc/xdg/parcellite/parcelliterc'

I've googled and found some relevant matches for the `Flag ... status... EXIT... STAT 0` thing, but nothing with solutions that I could apply.

I believe there is a UI, but I don't know where it is hiding :sob: Thanks for overtaking this project, I'd really appreciate if you could help.

**Comment by junkyardsparkle** [comment]
By default, the keybinding for the menu is <Ctrl><Alt>P, did you try that? You can get to the preferences from there.

**Comment by sverhagen** [comment]
Funny, I was trying to extract a **p**arameter in IntelliJ, this afternoon and got a very unusual menu. There was, indeed, Parcellite. So, got that covered, like you said. Thank you for the reply. Less important, but do you have any suggestion of how to get the icon to show up in the top of my screen? I suppose that's the intended behavior.

**Comment by junkyardsparkle** [comment]
No idea, sorry. I use it with Fluxbox window manager, launching it with the -n option specifically to keep the taskbar icon from showing up. I use it via keybindings, which have been changed to use <Mod4> ("Windows key") along with other keys that don't conflict with any other globals.

**Comment by rickyrockrat** [comment]
The icon should show up in the systray, assuming Ubuntu still has that. If it isn't working, can you post the version you are running? Thanks.

**Comment by sverhagen** [comment]
Ubuntu 18.04.2 LTS.

I assume systray is the thing in the top, like this, which is where other icons appear to show up:

![image](https://user-images.githubusercontent.com/2049860/53787048-67616080-3ed2-11e9-8231-acf00e21a6bf.png)

Thanks.

**Comment by rickyrockrat** [comment]
Yes, it is. I assume parcellite was running when you took that screenshot? Can you dump the output of this command here?
ps aux|grep indicator
Thanks.

**Comment by sverhagen** [comment]
ps aux|grep indicator

That returns nothing (other than the `grep` process itself).

**Comment by rickyrockrat** [comment]
This is why it is not showing up. parcellite looks for the indicator service (which apparently Ubuntu has done away with). I'll have to figure out another way to determine Parcellite is running Ubuntu.
Thanks.

**Comment by DSLituiev** [comment]
i have the same issue: no GUI visible on Ubuntu18  (`<Ctrl><Alt>P` helped -- but I had to find this thread) -- maybe including this instruction in `-h` manual will be an easy fix? or a cmd line parameter like `parcellite -g --gui`

**Comment by rickyrockrat** [comment]
OK, I have added a preference in commit 6d2585b. 

Can one of you build it and check it to see if it fixes the whole missing Icon thing? It is under Preferences->Display->Running unity.

**Comment by wez** [comment]
parcellite also does not show up in when running under Gnome/mutter on Fedora (Parcellite 1.2.1, GTK 2.24.33).  There is no `indicator` process running there either.

**Comment by MostHated** [comment]
I do not see it on Pop_OS 20.10 either. I thought it was not working because I was getting those same "Looking in '/etc/xdg/xdg-pop/parcellite/parcelliterc'" messages until I saw the post about the hotkey.

**Comment by rickyrockrat** [comment]
Can you please see if checking the Display->Running Unity fixes the icon not showing?
